templates/helpers.js

```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
```

Comment:
afaik packageObject.name is actually the name given in the `@page` tag on the closest parent documentation page with a `@package` tag